docs/source/api_reference/configuration/index.rst

```diff
@@ -0,0 +1,5 @@
+
+.. automodule:: pseudopeople.configuration.interface
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
Why is this here?  Shouldn't this be in develop already?

Reply:
I moved it into it's own "module".